rebar.config

```diff
@@ -3,7 +3,7 @@
             {i, "include"},
             {platform_define, "^2", binary_http_uri},
             {platform_define, "^2[1-9]", post20}]}.
-{minimum_otp_vsn, "18.0"}.
+{minimum_otp_vsn, "20.0"}.
```

Comment:
Even if some tests are failing in 19.x (assuming they're not failing "because of 19.x") would it be possible to try and keep this at 19.x?

Reply:
Updated to 19.0.